packages/core/src/devtools/store.ts

```diff
@@ -41,4 +41,7 @@ export const debugData = {
   getSequenceDetails: (sequenceHash: SequenceHash): typeof sequenceDetails[string] | undefined => {
     return sequenceDetails[sequenceHash];
   },
+  hasCSSRule: (rule: string): boolean => {
+    return cssRules.includes(rule);
+  },
```

Comment:
I am thinking to turn `cssRules` into `Set` if it's really needed:

```diff
-const cssRules: string[] = [];
+const cssRules = new Set<string>();
```

```diff
  addCSSRule: (rule: string) => {
-    cssRules.push(rule);
+    cssRules.add(rule);
  },
```

```diff
  getCSSRules: (): string[] => {
-   return cssRules;
+   return Array.from(cssRules);
  },
```

Reply:
Yes, this will solve the duplication problems and eliminate the need to check if the string exists.